keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/titus/TitusClusterHandler.kt

```diff
@@ -591,13 +594,13 @@ class TitusClusterHandler(
         digest = image.dockerImageDigest
       ),
       entryPoint = entryPoint,
-      resources = resources,
+      resources = resources.run { Resources(cpu, disk, gpu, memory, networkMbps) },
```

Comment:
Is this just making a copy of the `resources` object? What's the rationale for doing that here?

Reply:
It's converting to a different type